src/config/production.ts

```diff
@@ -23,7 +23,6 @@ const config: ConfigInterface = {
   sentry: {
     dsn: 'https://e257e8d6646e4657b4f556efc1de31e8@o1090662.ingest.sentry.io/6106929',
     environment: 'production',
-    release: process.env.SENTRY_RELEASE ?? throwError('No Sentry Release defined'),
```

Comment:
why does this go away?

Reply:
I moved it [here](https://github.com/iasql/iasql-engine/pull/1706/files#diff-198a1431d7c5e26ea78bc6e54b34b54e6f8ab342dd48ea11013877e8466a87c5R15) because now there's only one version in the codebase and so there's no longer any ambiguity on what version to send along to Sentry.